CMakeLists.txt

```diff
@@ -70,33 +68,196 @@ set(GLFW_BUILD_EXAMPLES OFF CACHE STRING "" FORCE)
+    Game/dmusic/track.cpp
+    Game/dmusic/wave.cpp
+    Game/game/aiouputpipe.cpp
+    Game/game/aistate.cpp
+    Game/game/definitions/spelldefinitions.cpp
+    Game/game/definitions/svmdefinitions.cpp
+    Game/game/definitions/cameradefinitions.cpp
+    Game/game/definitions/fightaidefinitions.cpp
+    Game/game/definitions/musicdefinitions.cpp
+    Game/game/definitions/particlesdefinitions.cpp
+    Game/game/definitions/sounddefinitions.cpp
+    Game/game/definitions/visualfxdefinitions.cpp
+    Game/game/questlog.cpp
+    Game/game/worldstatestorage.cpp
+    Game/game/fightalgo.cpp
+    Game/game/gamesession.cpp
+    Game/game/inventory.cpp
+    Game/game/movealgo.cpp
+    Game/game/playercontrol.cpp
+    Game/game/serialize.cpp
+    Game/game/gamescript.cpp
+    Game/graphics/abstractobjectsbucket.cpp
+    Game/graphics/attachbinder.cpp
+    Game/graphics/landscape.cpp
+    Game/graphics/light.cpp
+    Game/graphics/objectsbucket.cpp
+    Game/graphics/particlefx.cpp
+    Game/graphics/protomesh.cpp
+    Game/graphics/renderer.cpp
+    Game/graphics/sky/sky.cpp
+    Game/graphics/submesh/animmesh.cpp
+    Game/graphics/submesh/staticmesh.cpp
+    Game/graphics/ubostorage.cpp
+    Game/graphics/visualfx.cpp
+    Game/graphics/inventoryrenderer.cpp
+    Game/graphics/meshobjects.cpp
+    Game/graphics/animation.cpp
+    Game/graphics/animationsolver.cpp
+    Game/graphics/mdlvisual.cpp
+    Game/graphics/rendererstorage.cpp
+    Game/graphics/skeleton.cpp
+    Game/graphics/pfxobjects.cpp
+    Game/graphics/worldview.cpp
+    Game/graphics/pose.cpp
+    Game/main.cpp
+    Game/physics/dynamicworld.cpp
+    Game/physics/physicmesh.cpp
+    Game/physics/physicmeshshape.cpp
+    Game/ui/chapterscreen.cpp
+    Game/ui/dialogmenu.cpp
+    Game/ui/documentmenu.cpp
+    Game/ui/stacklayout.cpp
+    Game/ui/gamemenu.cpp
+    Game/ui/inventorymenu.cpp
+    Game/ui/menuroot.cpp
+    Game/utils/cp1251.cpp
+    Game/utils/inifile.cpp
+    Game/utils/workers.cpp
+    Game/utils/fileutil.cpp
+    Game/utils/installdetect.cpp
+    Game/utils/crashlog.cpp
+    Game/world/focus.cpp
+    Game/world/fplock.cpp
+    Game/world/gsoundeffect.cpp
+    Game/world/soundfx.cpp
+    Game/world/spaceindex.cpp
+    Game/world/triggers/abstracttrigger.cpp
+    Game/world/triggers/codemaster.cpp
+    Game/world/triggers/messagefilter.cpp
+    Game/world/triggers/trigger.cpp
+    Game/world/triggers/triggerlist.cpp
+    Game/world/triggers/triggerscript.cpp
+    Game/world/triggers/triggerworldstart.cpp
+    Game/world/triggers/zonetrigger.cpp
+    Game/world/triggers/movetrigger.cpp
+    Game/world/waymatrix.cpp
+    Game/world/waypath.cpp
+    Game/world/waypoint.cpp
+    Game/world/bullet.cpp
+    Game/world/item.cpp
+    Game/world/worldobjects.cpp
+    Game/world/interactive.cpp
+    Game/world/npc.cpp
+    Game/world/world.cpp
+    Game/world/worldsound.cpp
+    Game/camera.cpp
+    Game/gothic.cpp
+    Game/mainwindow.cpp
+    Game/resources.cpp)
+
+# shaders
```

Comment:
Since compilation of shader moved from *.bat, it make sense to move commentary for ubershading:
```
# Ubershader flags:
#   OBJ        - enable object matrix
#   SKINING    - animation skeleton
#   SHADOW_MAP - output is shadowmap
#   ATEST      - use alpha test
#   PFX        - use color modulation
```

Reply:
Sure thing.